cg/store/models.py

```diff
@@ -867,3 +868,23 @@ class SampleLaneSequencingMetrics(Model):
 
     def to_dict(self):
         return to_dict(model_instance=self)
+
+
+class NanoporeAcquisition(Model):
```

Comment:
What is an acquisition? Maybe NanoporeSequencingMetrics or similar is an alternative

Reply:
Yes that was the alternative. Acquisition is what it's called elsewhere in the files and documentation (by the company ONT). It's basically a run/analysis. Let's see what the others prefer and go with that